project1.html

```diff
@@ -2,40 +2,46 @@
 <head>
 	<link rel="stylesheet" type="text/css" href="project1.css">
 	<link rel="stylesheet" type="text/css" href="normalize.css">
+	
 </head>
 	<header>
 		<nav class="navbar">
 			<input class="nav-button" type="button" value="About">
 			<input class="nav-button" type="button" value="Portfolio">
+			<img src="liam.png" alt="myname">
 			<input class="nav-button" type="button" value="Contact">
+			<input class="nav-button" type="button" value="Download CV">
 		
 		</nav>
 	</header>
 <body>
 	<section class="intro-page">
-		<h1>liamlwrnc.</h1>
-		<h2>Hey I'm Liam. Welcome to my portfolio.</h2>
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
-	
+		<h2>Hey I'm Liam and that's Ronnie. Welcome to my portfolio.</h2>
+	</section>
 	
+	<section class="about-page">
+		<h1>About</h1>
+		<p>My name's Liam Lawrence and I'm a student at iO Academy.<br> 
+		   After working in the hospitality industry for 10 years I<br> 
+		   decided it was well and truly time for a change. <br>
+		   Constantly looking to learn and improve so for any <br>
+		   work opportunities or collab requests, please hit me <br>
+		   up through the Contact page. <br>
+		   <br>
+		   Peace and love.<br>
+		   Liam </p> 
+	</section>
 	
 	
 	
+	<section class="contact-page">
+		<h1>Contact</h1>
+		<button href="#email" type="button"><img src="emailicon.png"></button>
+		<button href="https://instagram.com/liamlwrnce" type="button"><img src="instaicon.png"></button>
+		<button type="button"><img src="linkedinicon.png"></button>
```

Comment:
these should probably also be `<a>` tags, since they are linking the user to another page